versions/h-/h5support.json

```diff
@@ -1,9 +1,9 @@
 {
   "versions": [
     {
-      "version": "1.0.7",
+      "version": "1.0.10",
       "port-version": 0,
-      "git-tree": "330266c4ab200c2d9c5e08397ef23d206939fc40"
+      "git-tree": "f18723051140ee1c440e13d457a5d078a2722de4"
```

Comment:
This should be in a new object.

Reply:
where does this git hash come from again?

Reply:
`git rev-parse HEAD:ports/h5support`